### ISSUE TITLE
cannot build swagger-core with Java 8

### ISSUE THREAD

**Issue by abstratt** [swagger-api/swagger-core#628]
Getting this trying to build swagger-core with Java 8 (more specifically the swagger-annotations module). Downgrading to Java 7 avoids this issue.

```
[ERROR] error: error while loading AnnotatedElement, class file '/home/rafael/dev/jdk1.8.0_05/jre/lib/rt.jar(java/lang/reflect/AnnotatedElement.class)' is broken
[INFO] (class java.lang.RuntimeException/bad constant pool tag 15 at byte 2713)
[ERROR] error: error while loading CharSequence, class file '/home/rafael/dev/jdk1.8.0_05/jre/lib/rt.jar(java/lang/CharSequence.class)' is broken
[INFO] (class java.lang.RuntimeException/bad constant pool tag 15 at byte 1501)
[ERROR] two errors found
```

**Comment by davidgev** [comment]
Hi Guys, is this fixed in the newer versions of Swagger?

**Comment by fehguy** [comment]
yes it is, fixed in #681.  Note, this was only to build locally, the artifacts worked fine.
